data/hai/hai jobs.txt

```diff
@@ -1503,3 +1503,214 @@ mission "Hai Prisoner Roleplay"
 		payment 15000 150
 		"reputation: Hai" += 5
 		dialog "The Hai depart your ship chittering in excitement. One of them hands you your payment of <payment> before running to catch up with their friends."
+
+
+
+mission "Swiftsong Business Convention [1]"
+	name "Transport to business convention at <planet>"
+	job
+	repeat
+	description "A group of <bunks> Hai who need to get to a business convention will pay you <payment> in exchange for transporting them and their <tons> of supplies to <planet>."
+	passengers 4 2 .1
+	cargo "office supplies" 1 2
+	to offer
+		random < 10
+	source
+		government "Hai"
+		not planet "Swiftsong"
+	destination "Swiftsong"
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+	on complete
+		payment 10000 500
+		"reputation: Hai" += 3
+		dialog `During the trip, your passengers appeared to be practicing a presentation for the conference. You sense a tinge of nervousness as they give you <payment> and leave your ship.`
+
+
+
+mission "Swiftsong Business Convention [2]"
+	name "Transport to business convention at <planet>"
+	job
+	repeat
+	description "A group of <bunks> Hai who need to get to a business convention will pay you <payment> in exchange for transporting them and their <tons> of supplies to <planet>."
+	passengers 4 4 .1
+	cargo "office supplies" 1 3
+	to offer
+		random < 5
+	source
+		government "Hai"
+		not planet "Swiftsong"
+	destination "Swiftsong"
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+	on complete
+		payment 10000 500
+		"reputation: Hai" += 3
+		dialog `During the trip, your passengers appeared to be practicing a presentation for the conference. You sense a tinge of nervousness as they give you <payment> and leave your ship.`
+
+
+
+mission "Swiftsong Film Festival [1]"
+	name "Transport to film festival at <planet>"
+	job
+	repeat
```

Comment:
For all of these, have the one-word tags first. We also tend to keep the name and description next to one another.
```suggestion
	job
	repeat
	name "Transport to film festival at <planet>"
```